CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 
 ### Changed
 
+### Fixed
+
+<a id="1_34_1"></a>
+
+## [1.34.1] - 2022-11-28
+
 ### Fixed
 - fix webxdc: allow `self` and `blob:` in `connect-src` in CSP
```

Comment:
imu, the changelogs are written for the end users, we also point from the device message there. so, we should use a much less technical language here. saying what it actually fixes for the user. for a technical view, we have the commit messages.

this is no blocker, more a general note. i also wanted to point that out before, and never came to it, and this line is just a good example :)

Reply:
so "fix some private apps", no not a good idea to remove all info, these entries are interesting for webxdc devs and they should know about CSP, other people can just ignore what they don't understand, it is a log not a marketing document.